example.py

```diff
@@ -1,29 +1,26 @@
 """
-Example chat bot that monitors incoming messages and sends "Hi!" every second.
+Example chat bot that monitors incoming messages. responds to !ping command
 """
 
+from tornado.ioloop import IOLoop
 from chatty import create
 import config
+import ChatEventHandler
 
-from tornado.ioloop import PeriodicCallback, IOLoop
-from functools import partial
+
+def _handle_chat(data):
+    ChatEventHandler.handler(data, CHAT)
 
 
 if __name__ == "__main__":
-    chat = create(config)
+    CHAT = create(config)
```

Comment:
I use VSCode for coding and the python extension I installed uses pylint, and only changed to stop it showing an error. I do check with flake8 before a commit